app.rb

```diff
@@ -0,0 +1,83 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  attr_accessor :people, :books, :rentals
+
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def list_people
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+
+  def create_person
+    print 'Do you want to Create a student (1) or a teacher (2)? [Input the number]: '
+    select_person = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Age: '
+    age = gets.chomp.to_i
+    case select_person
+    when 1
+      print 'Has parent permission? [Y/N]: '
+      permission = gets[0].capitalize
+      permission = permission == 'Y'
+      @people.push(Student.new(age, name, parent_permission: permission))
```

Comment:
- [ ] Good job so far but when we create a student the name doesn’t display it displays unknown instead of the original name kindly fix that 🙏🏾 

<img width="549" alt="Screenshot 2023-08-30 at 18 56 28" src="https://github.com/Zohra-Neda/OOP-School-Library/assets/74501523/dd2ea0bc-6d8f-449e-ae9b-11ec7e77f9a5">